code/modules/reagents/reagent_containers/chem_bag.dm

```diff
@@ -0,0 +1,70 @@
+/obj/item/reagent_containers/chem_bag
```

Comment:
I know chem bags and blood packs aren't that different, but I feel something off for merging them into one, so I made this seperated.